autowebcompat/utils.py

```diff
@@ -81,20 +81,12 @@ def prepare_images():
 
 
 def load_image(fname, parent_dir='data_resized'):
-    img = load_img(os.path.join(parent_dir, fname), target_size=(32, 24))
+    img = load_img(os.path.join(parent_dir, fname), target_size=(224, 224))
```

Comment:
Can you remove this change from the PR? It should be done in a separate PR, after this one.